src/functTest/groovy/com/bmuschko/gradle/docker/DockerWorkflowFunctionalTest.groovy

```diff
@@ -484,7 +484,7 @@ class DockerWorkflowFunctionalTest extends AbstractFunctionalTest {
     private File createDockerfile(File imageDir) {
         File dockerFile = new File(imageDir, 'Dockerfile')
         dockerFile << """
-FROM ubuntu:12.04
+FROM ubuntu:14.04
```

Comment:
I just wonder if we can use `alpine` to reduce download size and speed up things?